packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -106,7 +106,10 @@ const _SelectInput = (
         leadingIcon={icon}
         hasPopup={getActionListContainerRole(hasFooterAction, dropdownTriggerer)}
         isPopupExpanded={isOpen}
-        onClick={onTriggerClick}
+        onClick={(e) => {
+          onTriggerClick();
```

Comment:
Changing the default behaviour here? Earlier event would also get passed to `onTriggerClick`
```suggestion
          onTriggerClick(e);
```

Reply:
Oh wait, interesting, so in the typedefs we said that onTriggerClick can't accept any event argument. 

But previously how we set this prop, it bypassed the ts check. 

So I think this is fine.